web/src/components/data-fields/AccountSetupForm.tsx

```diff
@@ -20,7 +20,7 @@ interface Props extends FormContextFieldProps {
   setUser: (user: BusinessUser) => void;
 }
 
-export const NameAndEmail = (props: Props): ReactElement => {
+export const AccountSetupForm = (props: Props): ReactElement => {
```

Comment:
Renamed this component and file to more accurately capture what is inside- previously was just called `NameAndEmail` even though it also contained data sharing consent boxes.